src/library_browser.js

```diff
@@ -27,6 +27,7 @@ var LibraryBrowser = {
 #endif
     Module["requestAnimationFrame"] = (func) => Browser.requestAnimationFrame(func);
     Module["setCanvasSize"] = (width, height, noUpdates) => Browser.setCanvasSize(width, height, noUpdates);
+    Module["setHiDPIAware"] = (isHiDPIAware) => Browser.setHiDPIAware(isHiDPIAware);
```

Comment:
Do we need to expose this on the outer `Module` object like this?

Would it make more sense to have this be a build time setting?  Or is this something one would want to change at runtime?

Reply:
* Since I removed the code that initializes the `Browser.isHiDPIAware` variable there needs to be a way to set it
* I don't believe it can be a build time setting because I do believe that the client code needs to set it based on the environment (and I was definitely planning to have my program offer the user the ability to toggle it if the performance are not satisfactory for example).

Reply:
Fair enough